### PR TITLE
#163391160 Implement handler that allows authenticated users fetch their own articles

### PR DESCRIPTION
**Pull request by rafmme**
#### What does this PR do?
This PR implements handlers for fetching articles of a user

#### Description of Task to be completed?
* Implement a handler to fetch one article of a user
* Implement a handler to fetch all articles of a user
* Implement filtering of the articles

#### How should this be manually tested?
* git clone the repository
* run npm install
* Use POSTMAN to make api request to `api/v1/user/articles`, `api/v1/myArticles/:slug`, `api/v1/myArticles?drafts`, `api/v1/myArticles?tags=example`, `api/v1/myArticles?published`

#### What are the relevant pivotal tracker stories?
[Get User's articles](https://www.pivotaltracker.com/story/show/163391160)

### CODE REVIEW

**Comment by tejiri4** [comment]
LGTM, yet.

**Comment by ebzeal** [comment]
LGTM yet

File: server/controllers/ArticleController.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Boolify } from 'node-boolify';
+import { Sequelize } from 'sequelize';
```
Comment (jesseinit) [inline server/controllers/ArticleController.js]: Hey Timi,
The Sequelize import is rather redundant as it is also part of `db` you imported from `../models`

Comment (rafmme) [reply server/controllers/ArticleController.js]: Thanks for the feedback, I have refactored it.

**Review by ebzeal — COMMENTED** [review]
Looks good

**Pull request merged**